packages/protocol-kit/scripts/generateTypechainFiles.ts

```diff
@@ -83,30 +83,33 @@ function moveTypechainFiles(inDir: string, outDir: string): void {
   })
 }
 
-const ethersV5 = 'ethers-v5'
+function generateTypes(version: string) {
```

Comment:
This `version` variable name can be confused with the version of a Safe, but it refers to the use of `ethers` and `web3` and their used versions.
I suggest using `typechainTarget`, which is the value expected here `typechain --target `